src/import/client.rs

```diff
@@ -218,6 +227,34 @@ impl ImportClient for Client {
     fn has_region_id(&self, id: u64) -> Result<bool> {
         Ok(self.pd.get_region_by_id(id).wait()?.is_some())
     }
+
+    fn is_scatter_region_finished(&self, region_id: u64) -> Result<bool> {
+        match self.pd.get_operator(region_id) {
+            Ok(resp) => {
+                // If the current operator of region is not `scatter-region`, we could assumption
+                // that `scatter-operator` has finished or timeout.
+                let scatter_region = "scatter-region";
+                if resp.kind.len() != scatter_region.len() || resp.kind != scatter_region.as_bytes()
+                {
+                    Ok(true)
+                } else {
+                    Ok(resp.status != OperatorStatus::RUNNING)
+                }
+            }
+            Err(PdError::RegionNotFound(_)) => Ok(true), // heartbeat may not send to PD
+            Err(err) => {
+                error!("check scatter region operater result"; "region id" => %region_id, "err" => %err);
```

Comment:
Please use `region_id` and `error`.

Reply:
addressed

Reply:
Seems `err` is more popular usage in TiKV.